org.citra.citra.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "org.citra.citra",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "citra-qt",
+    "rename-desktop-file": "citra.desktop",
+    "rename-icon": "citra",
+    "rename-appdata-file": "org.citra.citra.appdata.xml",
+    "build-options": {
+        "env": {
+            "CI": "1",
+            "GITHUB_ACTIONS": "1",
+            "GITHUB_REPOSITORY": "citra-emu/citra",
+            "GIT_TAG_NAME": "nightly-1694"
+        }
+    },
+    "finish-args": [
+        "--device=all",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--share=ipc",
+        "--filesystem=xdg-config/citra-emu:create",
+        "--filesystem=xdg-data/citra-emu:create",
```

Comment:
Technically you don't need these as long as Citra follows XDG_DATA_HOME and XDG_CONFIG_HOME environment variables.

Reply:
Yes, Citra uses Qt's platform API to determine the home directory. I will remove these lines.